components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/Nimbus.kt

```diff
@@ -616,8 +629,7 @@ open class Nimbus(
         activeExperiments.find { it.featureIds.contains(featureId) }?.also { experiment ->
             NimbusEvents.exposure.record(NimbusEvents.ExposureExtra(
                 experiment = experiment.slug,
-                branch = experiment.branchSlug,
-                enrollmentId = experiment.enrollmentId
+                branch = experiment.branchSlug
```

Comment:
Should the `featureId` be passed to the exposure extra constructor here?

Reply:
Yes, I neglected to add it back when updating the `enrollment_id` to `feature_id`. Another good catch!